File: api-project/MovieReservation/MovieReservation/DTO/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieReservation.DTO
{
    public class Schedule
    {
        public Schedule()
        {
            this.ShowBookingList = new List<Booking>();
        }
        public int ShowID { get; set; }
        public DateTime ShowTime { get; set; }
        public int ShowSeats { get; set; }
        public decimal ShowPrice { get; set; }
        public List<Booking> ShowBookingList { get; set; }
    }
}

File: angular-proj/src/app/DTO/Schedule.ts
import { Booking } from "./Booking";

export class Schedule {
    ShowID: number;
    ShowTime: Date;
    ShowSeats: number;
    ShowPrice: number;
    ShowBookingList: Array<Booking>;

    constructor(){
        this.ShowID = 0;
        this.ShowTime = new Date();
        this.ShowSeats = 18;
        this.ShowPrice = 0.00;
        this.ShowBookingList = new Array<Booking>();
    }
}